docs/controllers.rst

```diff
@@ -56,5 +56,14 @@ For the simple pattern of having a controller respond to an specific intent the
     return { reply: 'LaunchIntent.OpenResponse', to: 'die' };
   });
 
+If you receive a Display.ElementSelected type request, you could use the same approach for intents and state. Voxa receives this type of request and turns it into ``DisplayElementSelected`` Intent
+
+.. code-block:: javascript
+
+  skill.onIntent('DisplayElementSelected', (alexaEvent) => {
```

Comment:
I'm not really comfortable with our shadowing events and other types into intents. It's not really what the user would expect and somewhat breaks the mental-model of Alexa. Can we instead have a differently named handler for this.

Reply:
Amazon recommends this approach please take a look at this

> There is no built-in intent for selecting actions or list items. However, you can create an intent for this purpose and include it in the intent schema. This intent should be activated when the skill receives a Display.ElementSelected event in a response.

https://developer.amazon.com/docs/custom-skills/display-interface-reference.html#touch-selection-events